dvc/scm/base.py

```diff
@@ -7,6 +7,27 @@
 from dvc.exceptions import DvcException
 
 
+DIFF_TARGET = "target"
+DIFF_IS_DIR = "is_dir"
+DIFF_OLD_FILE = "old_file"
+DIFF_OLD_CHECKSUM = "old_checksum"
+DIFF_NEW_FILE = "new_file"
+DIFF_NEW_CHECKSUM = "new_checksum"
+DIFF_SIZE = "size_diff"
+DIFF_DEL = "del"
```

Comment:
These macros are not used in scm-logic at all, they are related to `dvc diff` itself, aren't they? If so, they should  not be here.

Reply:
I can put them in `dvc/repo/diff`, will it be ok, hat do you think?

Reply:
@django-kz Yes, that would be better.